Tests/LibWeb/Ref/root-rem.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<link rel="match" href="reference/root-rem.html" />
+<style>
+    :root {
+        font-size: 1.2rem;
+    }
+</style>
+<script>
+    const span = document.getElementById("message");
+    // Trigger redraw
+    internals.dispatchUserActivatedEvent(span, new Event("mousedown"));
```

Comment:
Not sure if this is good in a test because if we ever make our CSS re-compute criteria smarter, this test becomes useless. Is there a more future-proof way to do this? Should I add a comment about this?

Reply:
This call fails because `span` is null at this point. The `<span>` element you're referencing needs to appear before the `<script>` element.

FWIW, a lot of Web Platform Tests use `element.offsetTop` or `document.body.offsetTop` when they need to trigger a relayout. Probably not the most principled way to do it but it does seem to be widely used by WPT.